parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -110,6 +110,13 @@ configureEtcd() {
 ensureNTP() {
```

Comment:
We can get rid of this `ensureNTP` func definition entirely as the only place that calls it is being deleted